mypyc/test-data/irbuild-classes.test

```diff
@@ -2088,3 +2088,102 @@ class NonNative:
 @mypyc_attr(free_list_len=1, allow_interpreted_subclasses=True)  # E: "free_list_len" can't be used in a class that allows interpreted subclasses
 class InterpSub:
     pass
+
+[case testUnsupportedGetAttr]
+from mypy_extensions import mypyc_attr
+from typing import Optional
+
+@mypyc_attr(allow_interpreted_subclasses=True)
+class AllowsInterpreted:
+    def __getattr__(self, attr: str) -> Optional[object]:  # E: "__getattr__" not supported in class "AllowsInterpreted" because it allows interpreted subclasses
+        return 0
+
+class InheritsInterpreted(dict):
+    def __getattr__(self, attr: str) -> Optional[object]:  # E: "__getattr__" not supported in class "InheritsInterpreted" because it inherits from an interpreted class
+        return 0
+
+[case testGetAttr]
+from typing import Optional, Tuple
+
+class GetAttr:
+    class_var = "x"
```

Comment:
Test also a "real" class variable that is annotated using `ClassVar[t]`.

Reply:
obtaining the value through the wrapper works (since i've also added this to the run tests) but when the variable is annotated like this, mypyc seems to always call `CPyObject_GetAttr` instead of accessing it directly. not a regression since it works like this without `__getattr__` too but maybe it could be optimized in the future.